.github/workflows/release.yaml

```diff
@@ -0,0 +1,34 @@
+name: Release
+
+on:
+  create:
+    tags:
+      - 'v*'
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
+
+      - name: Get dependencies
+        run: make deps
+
+      - name: Build
+        run: |
+          GOOS=linux GOARCH=amd64 go build -o bin/tc-redirect-tap-amd64 ./cmd/tc-redirect-tap
+          GOOS=linux GOARCH=arm64 go build -o bin/tc-redirect-tap-arm64 ./cmd/tc-redirect-tap
+
+      - name: Release
+        uses: softprops/action-gh-release@v1
+        with:
```

Comment:
We're going to want release notes to go into the releases page. For firecracker-go-sdk, we have a CHANGELOG.md where we keep a running list of changes with a header for the version number. Maybe we could do the same here and parse out the set of changes to go in the release notes? (I think something simple like find the line that starts with `# ${RELEASE_VERSION}` and copy every line after that until the next line that starts with `#`.

That might be fine for a follow up PR, though.

Reply:
Knowing that there is at the moment no release file, I think it's more convenient to add this step to the workflow later in another PR to make it testable. What is your feeling about this ?

Reply:
We have that available and use it widely across all OpenFaaS projects and some of my own:

Have a look if the format is suitable for you? https://github.com/alexellis/arkade/releases

Reply:
I think that could be reduced to just the changes that would be visible to someone consuming the binary, but yes, something like that would work. 

Let's defer that discussion for now. I think we can get one release out first and then worry about how to structure future release notes.

Reply:
SGTM @Kern-- - when do you expect to merge and release a binary?

We can then wind down https://github.com/alexellis/tc-tap-redirect-builder

Reply:
I'll merge this now. We have a little more process to follow than just pushing a tag today, so I'll get it going now to get this out ASAP.